satellite/satellitedb/pbold/order.go

```diff
@@ -5,10 +5,11 @@ package pbold
 
 import (
 	fmt "fmt"
+	math "math"
```

Comment:
Do not change this file! See the header `// Code generated by protoc-gen-gogo. DO NOT EDIT.	`

Reply:
reverted